### PR TITLE
Add landing messages. #1

### PR DESCRIPTION
**Pull request by Drpampam**
I have completed the task and hope issue Is resolved

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <title>Page Title</title>
 </head>
 <body>
-
+    <h1>This is my first portion</h1>
```
Comment (johnOmotoye) [inline Index.html]: Could you change this and make it look more like a landing page?, "Welcome to Portionify" would have been a better choice

**Review by johnOmotoye — CHANGES_REQUESTED** [review]

File: Index.html (hunk 1 above)
Comment (Drpampam) [reply Index.html]: i will make changes and revert

Comment (Drpampam) [reply Index.html]: changes is made

**Review by johnOmotoye — APPROVED** [review]
Better, thanks

**Comment by johnOmotoye** [comment]
appreciated

**Pull request merged**